packages/web-config-server/src/apiV1/dataBuilders/transform.js

```diff
@@ -3,15 +3,20 @@
  * Copyright (c) 2019 Beyond Essential Systems Pty Ltd
  */
 
-const orgUnitCodeToName = async (models, value) => {
-  const organisationUnit = await models.entity.findOne({ code: value });
-  const { name } = organisationUnit || {};
-
-  return name || value;
+const entityFindNameByCondition = async (models, condition) => {
```

Comment:
Function names generally start with a verb, since they are "actions" taken. Let's also use `conditions` instead of `condition`:

```suggestion
const findEntityName = async (models, conditions) => {
```